metabrainz/templates/index/about.html

```diff
@@ -3,42 +3,44 @@
 {% block title %}{{ _('About') }} - MetaBrainz Foundation{% endblock %}
 
 {% block content %}
-  <h1 class="page-title">{{ _('About the MetaBrainz Foundation') }}</h1>
+  <h1 class="page-title"><img src="{{ url_for('static', filename='img/logos/metabrainz.svg') }}" class="hidden-xs"></h1>
+
+     <h2 class="page-title">{{ _('About the MetaBrainz Foundation') }}</h2>
 
   <div class="row">
     <div class="col-sm-9">
       <p>
-        {{ _('The MetaBrainz Foundation is a 501(c)(3) tax-exempt non-profit founded in
-        December 2004 and based in San Luis Obispo, California, that believes in
-        free, open access to data.') }}
+        {{ _('The MetaBrainz Foundation is a global community creating and maintaining 
+        an open encyclopedia of music and arts metadata. Beginning as a resource for music 
+        collectors we currently establish collaborations between volunteers, artists, and 
+        industry leaders. Since it’s founding it has grown into a global community of 
+        <a href="%(editors_url)s">hundreds of thousands of editors</a> 
+        collaborating together to build and maintain the MetaBrainz comprehensive datasets.') }}
       </p>
       <p>
-        {{ _('The MetaBrainz Foundation has been set up to build community maintained
-        databases and make them available in the public domain or under Creative
-        Commons licenses. Our data is mostly gathered by volunteers, and verified
-        by a voting system to make sure it is consistent and correct. All
-        non-commercial use is free, but commercial users are asked to pay for
-        regular updates of the data in order to help fund the project. We
-        encourage all data users to contribute to the data gathering process so
-        that our data can be as comprehensive as possible.') }}
+        {{ _('The foundation works within a balanced ecosystem of contributions to the datasets 
+        and community and the utilization of the datasets. We encourage all data users to participate 
+        in the data gathering, editing and peer review processes so that our data can be as comprehensive
+        as possible. Each entity is assigned a permanent 
+        <a href="%(MusicBrainz_Identifier_url)s">MBID (MusicBrainz Identifier)</a> 
```

Comment:
It's also a good idea to keep all names for placeholder values lowercase since:

1. They are basically python/jinja variable names.
2. It's easier for localisers not to worry about what case these names are in (since they are case sensitive) and just assume that they are lowercase by default.